src/taipy/gui/config.py

```diff
@@ -77,9 +77,9 @@
         "color_paper_dark": str,
         "font_family": str,
         "border_radius": int,
-        "input_button_height": str
+        "input_button_height": str,
```

Comment:
How come this additional (and useless) comma becomes mandatory?

Reply:
Can this be configured?